src/app/beer_garden/events/handlers.py

```diff
@@ -34,20 +34,23 @@ def garden_callbacks(event: Event) -> None:
         logger.debug(f"{event!r}")
 
     # These are all the MAIN PROCESS subsystems that care about events
-    for handler in [
-        beer_garden.application.handle_event,
-        beer_garden.garden.handle_event,
-        beer_garden.plugin.handle_event,
-        beer_garden.requests.handle_event,
-        beer_garden.requests.handle_wait_events,
-        beer_garden.router.handle_event,
-        beer_garden.systems.handle_event,
-        beer_garden.scheduler.handle_event,
-        beer_garden.log.handle_event,
-        beer_garden.files.handle_event,
-        beer_garden.local_plugins.manager.handle_event,
+    for handler, handler_tag in [
```

Comment:
This is a leftover from a previous try. It just makes the error message dumped to the logs be more user friendly.